### PR TITLE
Update server landing page for 20.04

### LINKED ISSUE #7193: Update the /server overview page
Update [Ubuntu Server - for scale out workloads](https://ubuntu.com/server) page baed on the [copy doc](https://docs.google.com/document/d/1vxLhtbjgPs9PCNEhr7fbhEoErCPMXYFfU4UTKH7Kbq4/edit)

### PR DESCRIPTION
**Pull request by deadlight**
## Done

- Updated /server landing page

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/server
- Compare with the [copydoc](https://docs.google.com/document/d/1vxLhtbjgPs9PCNEhr7fbhEoErCPMXYFfU4UTKH7Kbq4/edit#)
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)

## Issue / Card

Fixes #7193

### CODE REVIEW

File: templates/server/index.html [hunk 1]
```diff
@@ -40,18 +40,19 @@ <h2>
...
-      <li class="p-list__item is-ticked">LXD 3.0 - Linux containers including clustering, QoS, and resource controls (CPU, memory, block I/O, graphics, and storage quota)</li>
-      <li class="p-list__item is-ticked">Install snaps for simple application installation and release management</li>
-      <li class="p-list__item is-ticked">Linux 4.15 kernel</li>
-      <li class="p-list__item is-ticked">Certification as a guest on AWS, Microsoft Azure, Joyent, IBM, Google Cloud Platform and Rackspace</li>
-      <li class="p-list__item is-ticked">Updates to LXD (v3.0), DPDK (v17.11.1), Postgresql (v10.3), Libvirt (v4.0), NGINX (v1.14), Qemu (v2.11.1), Docker (v17.03), Puppet (v4.10), MySQL (v5.7), PHP (v7.2), and more</li>
+      <li class="p-list__item is-ticked">Supported by Canonical until 2025 with ESM coverage until 2030</li>
+      <li class="p-list__item is-ticked">Runs on all major architectures &ndash; x86-64, ARM v7, ARM64, POWER8, POWER9, IBM s390x (LinuxONE) and introducing initial support for RISC-V</li>
+      <li class="p-list__item is-ticked">Ubuntu Pro cloud images for AWS and Azure, which include hardening, certification, kernel livepatch and more</li>
```
Comment (pmahnke) [inline templates/server/index.html:45]: meant to be links

File: templates/server/index.html [hunk 2]
```diff
@@ -40,18 +40,19 @@ <h2>
...
+      <li class="p-list__item is-ticked">The Ubuntu Server Live installer is now able to update itself when connected to the internet for the latest features and bug fixes. Initial support for automated installs is now available as well.</li>
+      <li class="p-list__item is-ticked">SSH that supports enabling two factor authorization (2FA)</li>
```
Comment (pmahnke) [inline templates/server/index.html:47]: two-factor (with a dash) I added that

File: templates/server/index.html [hunk 3]
```diff
@@ -40,18 +40,19 @@ <h2>
...
+      <li class="p-list__item is-ticked">Uses nftables as the default engine for UFW which provides easier syntax and achieves better performance results than iptables</li>
+      <li class="p-list__item is-ticked">WireGuard<sup>&reg;</sup> &ndash; an innovative VPN technology with modern cryptography defaults and streamlined usability</li>
+      <li class="p-list__item is-ticked">AppArmor3 for an even more secure system</li>
+      <li class="p-list__item is-ticked">More resilient bootloader that tolerates disk failures</li>
+      <li class="p-list__item is-ticked">Greater support for IPv6 on Microsoft Azure</li>
+      <li class="p-list__item is-ticked">Support for latest Instance Metadata Service (IMDSv2) on Amazon Web Services (AWS)</li>
+      <li class="p-list__item is-ticked">The latest longterm Linux 5.4 kernel for the latest hardware and security updates</li>
```
Comment (pmahnke) [inline templates/server/index.html:54]: should be 'long-term'

**Comment by pmahnke** [comment]
@deadlight sorry I clicked a few times and you get repeat 'and please delete 19.10 section'

**Comment by deadlight** [comment]
@pmahnke fixed

**Pull request merged**